compiler/damlc/tests/daml-test-files/SubmitterInMaintainers_before_1866.daml

```diff
@@ -1,95 +0,0 @@
--- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
--- SPDX-License-Identifier: Apache-2.0
-
--- @UNTIL-LF 1.5
```

Comment:
Maybe we should error in the integration tests for files with `UNTIL-LF` annotations that are not supported :thinking:

Reply:
Yeah, I had the same thought. I'll put that in another PR. (Contribution stats, you know?)